app/public/css/styles.css

```diff
@@ -1175,7 +1175,8 @@ p {
   line-height: 32px;
   font-weight: bold;
   border: none;
-  transition: background-color var(--transition-easing) var(--transition-time);
+  transition: background-color var(--transition-easing) var(--transition-time),
+    color var(--transition-easing) var(--transition-time);
```

Comment:
It's more annoying that it changes colors out of sync.
Rather remove color transition.

![2019-01-15 13-07-16 2019-01-15 13_09_08](https://user-images.githubusercontent.com/2865988/51179755-c3dad300-18c6-11e9-8ec8-d2373e0f1578.gif)